_dashboards/im-dashboards/notifications.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Notification settings
+parent: Index management in Dashboards
+nav_order: 60
+---
+
+# Notification settings
+
+You can configure default notification settings for index operations globally on the **Notification settings** page. You can also configure additional notification settings for individual index operations.
+
+## Configuring default notification settings
+
+On the **Notification settings** page, you can configure the default notification settings for the following index operations that may take longer time to complete:
```

Comment:
This reads as though it should be two sentences, with the first sentence ending at "operations". Also, that may take longer than what (complete the implied comparison)?